src/NServiceBus.Persistence.DynamoDB.Tests/InstallerTests.cs

```diff
@@ -8,222 +8,129 @@
-
-                sagaSettings.PartitionKeyName = "PK2";
-                await installer.CreateSagaTableIfNotExists(sagaSettings, CancellationToken.None);
-                var table2 = await dynamoClient.DescribeTableAsync(sagaSettings.TableName);
-
-                Assert.AreEqual(table1.Table.CreationDateTime, table2.Table.CreationDateTime);
-                Assert.AreEqual("PK1", table1.Table.KeySchema[0].AttributeName);
-                Assert.AreEqual("PK1", table2.Table.KeySchema[0].AttributeName);
+                TestContext.WriteLine($"Error deleting queue: {e}");
             }
         }
 
-        class OutboxInstallationTests : InstallerTests
+        [Test]
+        public async Task Should_create_table_with_configured_attribute_names()
         {
-            [TearDown]
-            public async Task Teardown()
-            {
-                try
-                {
-                    await dynamoClient.DeleteTableAsync(outboxSettings.TableName);
-                }
-                catch (Exception e)
-                {
-                    TestContext.WriteLine($"Error deleting queue: {e}");
-                }
-            }
-
-            [Test]
-            public async Task Should_create_table_with_configured_attribute_names()
-            {
-                outboxSettings.PartitionKeyName = Guid.NewGuid().ToString("N");
-                outboxSettings.SortKeyName = Guid.NewGuid().ToString("N");
+            tableConfiguration.PartitionKeyName = Guid.NewGuid().ToString("N");
+            tableConfiguration.SortKeyName = Guid.NewGuid().ToString("N");
 
-                await installer.CreateOutboxTableIfNotExists(outboxSettings, CancellationToken.None);
+            await installer.CreateTable(tableConfiguration, CancellationToken.None);
 
-                var table = await dynamoClient.DescribeTableAsync(outboxSettings.TableName);
+            var table = await dynamoClient.DescribeTableAsync(tableConfiguration.TableName);
 
-                Assert.AreEqual(outboxSettings.TableName, table.Table.TableName);
-                Assert.AreEqual(outboxSettings.PartitionKeyName, table.Table.KeySchema[0].AttributeName);
-                Assert.AreEqual(KeyType.HASH, table.Table.KeySchema[0].KeyType);
-                Assert.AreEqual(ScalarAttributeType.S, table.Table.AttributeDefinitions.Single(a => a.AttributeName == outboxSettings.PartitionKeyName).AttributeType);
-                Assert.AreEqual(outboxSettings.SortKeyName, table.Table.KeySchema[1].AttributeName);
-                Assert.AreEqual(KeyType.RANGE, table.Table.KeySchema[1].KeyType);
-                Assert.AreEqual(ScalarAttributeType.S, table.Table.AttributeDefinitions.Single(a => a.AttributeName == outboxSettings.SortKeyName).AttributeType);
-                Assert.AreEqual(TableStatus.ACTIVE, table.Table.TableStatus);
-            }
+            Assert.AreEqual(tableConfiguration.TableName, table.Table.TableName);
```

Comment:
Again nothing needs to change here, just a thought.

Wonder if an approval test would be nicer by approving the table that was described (after scraping the random elements).